<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QLineEdit" name="writeDataInPortLineEdit">
      <property name="text">
       <string>000BDF2F2F4A32</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="openClosePortHorizontalLayout">
      <item>
       <widget class="QPushButton" name="openPushButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closePushButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="writeHorizontalLayout">
      <item>
       <widget class="QPushButton" name="writePushButton">
        <property name="text">
         <string>Write</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="writeHorizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="readLabel">
        <property name="text">
         <string>Read byte array: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="readValueLabel">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>381</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" rowspan="5">
     <widget class="QGroupBox" name="setupPortGroupBox">
      <property name="title">
       <string>Setup port</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="portsVerticalLayout">
         <item>
          <layout class="QHBoxLayout" name="listPortsHorizontalLayout">
           <item>
            <widget class="QLabel" name="portsLabel">
             <property name="text">
              <string>List ports: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="portsComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="manufacturerHorizontalLayout">
           <item>
            <widget class="QLabel" name="manufacturerLabel">
             <property name="text">
              <string>Manufacturer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="manufacturerValueLabel">
             <property name="minimumSize">
              <size>
               <width>113</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="descriptionHorizontalLayout">
           <item>
            <widget class="QLabel" name="descriptionLabel">
             <property name="text">
              <string>Description: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="descriptionValueLabel">
             <property name="minimumSize">
              <size>
               <width>113</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="serialNumberHorizontalLayout">
           <item>
            <widget class="QLabel" name="serialNumberLabel">
             <property name="text">
              <string>Serial Number: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="serialNumberValueLabel">
             <property name="minimumSize">
              <size>
               <width>113</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="vendorIdentifierHorizontalLayout">
           <item>
            <widget class="QLabel" name="vendorIdentifierLabel">
             <property name="text">
              <string>Vendor Identifier: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="vendorIdentifierValueLabel">
             <property name="minimumSize">
              <size>
               <width>113</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="productIdentifierHorizontalLayout">
           <item>
            <widget class="QLabel" name="productIdentifierLabel">
             <property name="text">
              <string>Product Identifier: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="productIdentifierValueLabel">
             <property name="minimumSize">
              <size>
               <width>113</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="systemLocationHorizontalLayout">
           <item>
            <widget class="QLabel" name="systemLocationLabel">
             <property name="text">
              <string>System Location: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="systemLocationValueLabel">
             <property name="minimumSize">
              <size>
               <width>113</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Box</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="baudRateHorizontalLayout">
         <item>
          <widget class="QLabel" name="baudRateLabel">
           <property name="text">
            <string>Baud Rate: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="baudRateComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>114</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="dataBitsHorizontalLayout">
         <item>
          <widget class="QLabel" name="dataBitsLabel">
           <property name="text">
            <string>Data bits: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dataBitsComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>114</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="parityHorizontalLayout">
         <item>
          <widget class="QLabel" name="parityLabel">
           <property name="text">
            <string>Parity: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="parityComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>114</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="stopBitsHorizontalLayout">
         <item>
          <widget class="QLabel" name="stopBitsLabel">
           <property name="text">
            <string>Stop bits: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="stopBitsComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>114</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="flowControlHorizontalLayout">
         <item>
          <widget class="QLabel" name="flowControlLabel">
           <property name="text">
            <string>Flow control: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="flowControlComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>114</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="savePushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="saveHorizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Exit the program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
